scripts/standardising.py

```diff
@@ -59,7 +59,11 @@ def main() -> None:
     parser.add_argument("--source", dest="source", nargs="+", required=True)
     arguments = parser.parse_args()
 
-    source = format_source(arguments.source)
+    try:
+        source = format_source(arguments.source)
+    except Exception as e:
```

Comment:
what happens if we dont catch it here? its already logged, so the CLI will just die right? what is the desired behaviour?

Reply:
Probably not throwing an exception to the user :)

Reply:
I think a bunch of that main function could throw straight back to the client.

I would generally have something like

```python

try:
  main()
except Exception as e:
  log_exception(e)
  exit(1)
```

where your logging all exceptions, and then exiting, how does `argparse` suggest you do error handling in its CLIs?

Reply:
> how does argparse suggest you do error handling in its CLIs?

[argparse Invalid arguments](https://docs.python.org/3/library/argparse.html#invalid-arguments)
Looks like they do something similar?